packages/jest-docblock/src/index.js

```diff
@@ -26,6 +26,12 @@ export function extract(contents: string): string {
   return match ? match[0].replace(ltrimRe, '') || '' : '';
 }
 
+export function strip(contents: string) {
+  const match = contents.match(docblockRe);
+  const docblockLength = match && match[0] ? match[0].length + 1 : 0;
+  return contents.substring(docblockLength);
```

Comment:
Could we do:

```
return match && match[0] ? contents.substring(match[0].length + 1) : contents;
```

instead?

Reply:
Good thinking! Better to return `contents` than to return `contents.substring(0)`